src/classes/Foo.cls

```diff
@@ -0,0 +1,6 @@
+public without sharing class Foo {
+    void foo() {
+        HttpRequest req = new HttpRequest();
+        req.setEndpoint('http://localhost:com');
```

Comment:
This endpoint is insecure. Please ensure all callouts are performed against secure endpoints.